src/lib/PostPage.svelte

```diff
@@ -15,8 +16,11 @@
   postRenderer.link = (href, title, text) => {
     return `<a href="${href}" title="${title}" rel="external">${text}</a>`
   }
+
+  const emtpyRenderer = new marked.Renderer()
```

Comment:
What is this for?

Reply:
Removed

Reply:
Something tells me Marked already provides something like that.